### PR TITLE
Caiman

### PR DESCRIPTION
**Pull request by 4D42**
- use print error message from try: except:
- change some variable names
- use `import  utilities as utils` in `caiman_utilities.py`

### CODE REVIEW

File: CaImAn/caiman_run.py [hunk 1]
```diff
@@ -54,17 +54,24 @@
 except SyntaxError:
     utils.print_to_intercept(ADVANCED_BAD_TYPE)
     sys.exit()
+except Exception as error:
+    #utils.print_error(error)
```
Comment (doriclenses) [inline CaImAn/caiman_run.py:58]: Should be commented?

File: CaImAn/caiman_run.py [hunk 2]
```diff
@@ -151,6 +158,9 @@
         except TypeError:
             utils.print_to_intercept(PARAM_WRONG_TYPE)
             sys.exit()
+        except Exception as error:
+            utils.print_error(error, MOTION_CORREC)
```
Comment (doriclenses) [inline CaImAn/caiman_run.py:162]: Can we change Motion correction text from capital letters?

**Review by doriclenses — CHANGES_REQUESTED** [review]

File: CaImAn/caiman_run.py (hunk 2 above)
Comment (4D42) [reply CaImAn/caiman_run.py:162]: yeap!

File: CaImAn/caiman_run.py (hunk 1 above)
Comment (4D42) [reply CaImAn/caiman_run.py:58]: good question and I don't know..... I want to add a message with it but I don't know what message... :(

Comment (doriclenses) [reply CaImAn/caiman_run.py:58]: 🤷‍♀️

**Review by doriclenses — APPROVED** [review]

**Review by kapilseh — APPROVED** [review]

**Pull request merged**